mlflow/pyfunc/__init__.py

```diff
@@ -2249,14 +2279,10 @@ def predict(model_input: List[str]) -> List[str]:
     if first_argument_set_specified and second_argument_set_specified:
         raise MlflowException(
             message=(
-                "The following sets of parameters cannot be specified together: {first_set_keys}"
-                " and {second_set_keys}. All parameters in one set must be `None`. Instead, found"
-                " the following values: {first_set_entries} and {second_set_entries}".format(
-                    first_set_keys=first_argument_set.keys(),
-                    second_set_keys=second_argument_set.keys(),
-                    first_set_entries=first_argument_set,
-                    second_set_entries=second_argument_set,
-                )
+                f"The following sets of parameters cannot be specified together:"
```

Comment:
It looks like you're using a newer version of ruff. `0.2.1` should be used.

https://github.com/mlflow/mlflow/blob/c85ff32270a9879d6be9851505fadaee4824a1de/requirements/lint-requirements.txt#L1

Reply:
I remember this is some lint tests telling me these lines are too long and I manually fixed them to pass those tests.